src/components/NewOrder.js

```diff
@@ -1,26 +1,76 @@
 import { Component, DOM as dom } from 'react';
 import { connect } from 'react-redux';
 import * as actions from '../action_creators';
+import _ from 'underscore';
 
 const mapStateToProps = function (state) {
   return {
-    orders: state.orders,
     items: state.items
```

Comment:
What do you think of renaming this collection to `masterItems` or something? Might get confusing to distinguish vs. `state.items`.